lib/std/tar.zig

```diff
@@ -894,6 +894,11 @@ test "header parse mode" {
 }
 
 test "create file and symlink" {
+    // With test enabled this is hanging under windows:
+    // zig build test docs --zig-lib-dir .\lib\ -Dstatic-llvm -Dskip-non-native -Denable-symlinks-windows
+    const builtin = @import("builtin");
+    if (builtin.os.tag == .windows) return error.SkipZigTest;
```

Comment:
When disabling a test, please first file an issue, then link to the issue when disabling it.

Reply:
I traced problem a little bit more and file the [issue](#19233).
I fixed this test. The problem was that created files were not closed.